dark/index.html

```diff
@@ -281,11 +311,58 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
       const menu_toggle = document.querySelector(".menu-toggle");
       const sidebar = document.querySelector(".sidebar");
 
+
       menu_toggle.addEventListener("click", () => {
         menu_toggle.classList.toggle("is-active");
         sidebar.classList.toggle("is-active");
       });
     </script>
-</body>
 
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+
+  <!-- Internal JavaScript -->
+  
+  <script>
+    // Auto Typing Effect Script
+
+    var typed = new Typed(".auto-type", {
+      strings: [
+        "STUDENT",
+        "DEVELOPER",
+        "PROGRAMMER",
+        "DESIGNER",
+        "EDITOR",
+        "PHOTOGRAPHER",
+        "FREELANCER",
+      ],
+      typeSpeed: 100,
+      backSpeed: 100,
+      loop: true,
+    });
+
+    
+    // Navigation Bar Scroll Animation
+    
+    $(window).scroll(function () {
+      if ($(document).scrollTop() > 400) {
+        $(".nav-bar").addClass("affix");
+        console.log("OK");
+      } else {
+        $(".nav-bar").removeClass("affix");
+      }
+    });
+
+    
+    // Mobile View Navigation Bar Menu Icon
+    
+    const menu_toggle = document.querySelector(".menu-toggle");
+    const sidebar = document.querySelector(".sidebar");
+
+    menu_toggle.addEventListener("click", () => {
+      menu_toggle.classList.toggle("is-active");
+      sidebar.classList.toggle("is-active");
+    });
+  </script>
+
+</body>
 </html>
```

Comment:
Add the end of the line @Rishav1707 
```suggestion
</html>

```